template/src/config/ui/colors.ts

```diff
@@ -8,6 +8,7 @@ enum Palette {
   ANGRY = '#dd3333',
   DEEP_PURPLE = '#5D2555',
   TRANSPARENT = 'rgba(0, 0, 0, 0)',
+  TRANSPARENT_DISABLED_BUTTON = 'rgba(0,0,1,0.14)',
```

Comment:
Let's not tie the names to how they might be used. Note that we have `Palette.ANGRY` not `Palette.ERROR_MESSAGE`.

Reply:
Thanks for noticing, renamed